src/Views/Shared/AlertListCompact.elm

```diff
@@ -0,0 +1,22 @@
+module Views.Shared.AlertListCompact exposing (view)
+
+import Alerts.Types exposing (AlertGroup)
+import Html exposing (Html, ul)
+import Html.Attributes exposing (class)
+import Views.Shared.AlertCompact
+
+
+view : AlertGroup -> Html msg
+view ag =
+    let
+        alerts =
+            List.concatMap
+                (\b ->
+                    b.alerts
+                )
+                ag.blocks
+    in
+        ul
+            [ class "list pa0"
+            ]
+            (List.indexedMap Views.Shared.AlertCompact.view alerts)
```

Comment:
This is beautiful code!! Forget my comment above. Changed with a5d772752b29a41aaec2fcf798b4e13a8eb3cf1f